lizrd/text/tokenizers.py

```diff
@@ -37,12 +38,16 @@ def text_to_ids(self, text: str) -> List[int]:
 
 
 class GPTTokenizer(AbstractTokenizer):
-    VOCAB_SIZE = 50257
```

Comment:
waaaait, czy my więc zmieniamy wszystkie modele? to mi się nie podoba jakoś za bardzo

Reply:
czy nie ma w tym vocabie juz jakiś specjalnych tokenów które można użyć, maybe?

Reply:
no, ale jeśli trzeba to spoko. Trochę się jednak boję o kompatybilność wsteczną...